tests/test_emails.py

```diff
@@ -12,8 +12,7 @@
 
 def test_get_email_base_context(site_settings):
     site = site_settings.site
-    logo_url = build_absolute_uri(
-        location=None) + static('images/logo-document.svg')
+    logo_url = build_absolute_uri(static('images/logo-document.svg'))
```

Comment:
Could you, if you have some spare time, add a test to ensure the S3 bucket case is correctly handled? This would fix on of the issue mentioned in #1999.

That would be great :tada:

Reply:
@NyanKiyoshi I read this issue but I'm not pretty sure how can I test it. Could you explain what exactly should be tested?

Reply:
Here is an example of a test testing the set-up of a S3 bucket:
https://github.com/mirumee/saleor/blob/cefc896d18aba433c5baee92345760e0a4ed5397/tests/test_core.py#L240-L246

Reply:
We could just mock `static` to return a full URL instead of presetting the whole S3 bucket